src/evict/evict_page.c

```diff
@@ -622,13 +624,13 @@ __evict_review(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t evict_flags, bool
 
     if (closing)
         LF_SET(WT_REC_VISIBILITY_ERR);
-    else if (F_ISSET(ref, WT_REF_FLAG_INTERNAL) || WT_IS_HS(S2BT(session)))
+    else if (F_ISSET(ref, WT_REF_FLAG_INTERNAL) || WT_IS_HS(btree))
```

Comment:
It's a bigger change, but we should make `WT_IS_HS` and `WT_IS_METADATA` accept the same type of handle, and that should be a `dhandle`. Peel that off into another ticket if you'd like.

Reply:
I agree, I will create a separate ticket to address that.

Reply:
Created WT-6991